concent_api/verifier/tasks.py

```diff
@@ -44,18 +44,29 @@
 def blender_verification_order(
     subtask_id: str,
     source_package_path: str,
+    source_size: int,
+    source_package_hash: str,
     result_package_path: str,
+    result_size: int,
+    result_package_hash: str,
     output_format: BlenderSubtaskDefinition.OutputFormat,  # pylint: disable=unused-argument
     scene_file: str,  # pylint: disable=unused-argument
-    report_computed_task: message.ReportComputedTask,
 ):
     assert source_package_path != result_package_path
+    assert source_package_hash != result_package_hash
+    assert (source_size and source_package_hash and source_package_path) or (result_size and result_package_hash and result_package_path)
```

Comment:
Actually here all elements must be present.

Reply:
Done